src/helper-modules/bpmn-helper/src/getters.js

```diff
@@ -566,7 +567,7 @@ async function getProcessConstraints(bpmn) {
  * and its name and description for human identification
  *
  * @param {(string|object)} bpmn - the process definition as XML string or BPMN-Moddle Object
- * @returns { Promise.<{ id: String, originalId: String, processIds: String[], name: String, description: String }> } object containing the identifying information
+ * @returns { Promise.<{ id: string, originalId: string, processIds: string[], name: string, description: string }> } object containing the identifying information
```

Comment:
Shouldn't `originalId` be optional since it might not exist in some/most definitions?